src/Components/Publishing/Header/Header.tsx

```diff
@@ -58,13 +62,15 @@ export const Header: React.SFC<HeaderProps> = props => {
         leadParagraph={leadParagraph}
       />
     )
-    // Feature
-  } else {
-    const deck = getDeck(article, children)
-    const vertical = getVertical(article, children)
-    const image = children && children[3]
+  }
 
-    if (article.layout === "feature") {
+  const deck = getDeck(article, children)
+  const vertical = getVertical(article, children)
+  const image = children && children[3]
+
+  // tslint:disable-next-line:switch-default
```

Comment:
Before, I guess the `default` case was standard - does that sound right or does a `default`-less switch case make sense now that there are more `layout` possibilities?

Reply:
Either works for me, I think it makes it more clear now to use `switch` so 👍.